src/AWS.Deploy.CLI/ServerMode/Controllers/DeploymentController.cs

```diff
@@ -615,13 +615,14 @@ public IActionResult GetDeploymentStatus(string sessionId)
                 if (state.DeploymentTask.Exception != null)
                 {
                     var innerException = state.DeploymentTask.Exception.InnerException;
+                    var message = innerException.GetTruncatedErrorMessage();
                     if (innerException is DeployToolException deployToolException)
                     {
-                        output.Exception = new DeployToolExceptionSummary(deployToolException.ErrorCode.ToString(), deployToolException.Message, deployToolException.ProcessExitCode);
+                        output.Exception = new DeployToolExceptionSummary(deployToolException.ErrorCode.ToString(), message, deployToolException.ProcessExitCode);
```

Comment:
At this point, it looks like the only variable thing is the error code. You could assign `output.Exception` after the if/else clause

Reply:
We also add the process exit code based on the if/else clause. I prefer to eave this as is for now.